test/build.info

```diff
@@ -266,10 +266,6 @@ IF[{- !$disabled{tests} -}]
   INCLUDE[srptest]=../include ../apps/include
   DEPEND[srptest]=../libcrypto libtestutil.a
 
-  SOURCE[memleaktest]=memleaktest.c
-  INCLUDE[memleaktest]=../include ../apps/include
-  DEPEND[memleaktest]=../libcrypto libtestutil.a
-
```

Comment:
Why remove this? It's still used with asan/lsan

Reply:
I could not get it to pass in a regular build and it doesn't seem needed given we now do leak detection in everything.

Reply:
It still tests leak detection, i.e. instead of testing crypto-mdebug, it tests that asan / lsan actually does its job corrctly on platform where we can expect it.

Reply:
... or well, as soon as #9294 is merged :wink:

Reply:
Once #9294 is merged I will bring back memleak.

Reply:
okay, it's back now (no need to wait)